01-introduction.Rmd

```diff
@@ -158,121 +167,140 @@ Because R is open source, these developments can easily build on previous work,
+But soon **sp** provided methods for advanced map making using both the base and lattice plotting system. 
+Despite this, a demand for the layered grammar of graphics was growing especially after the release of **ggplot2** in 2007.
+**ggmap** extended **ggplot2**'s spatial capabilities [@kahle_ggmap:_2013].
+However, its main purpose was the easy access of several APIs to automatically download map tiles (among others, GoogleMaps, OpenStreetmap) and subsequent plotting of these as a basemap. 
+<!--Additionally, *ggmap** lets you use (mainly Google's) geocoding and routing services.-->
+Though **ggmap** facilitated map-making with **ggplot2**, one main limitation remained.
+To make spatial data work with the ggplot system, one needed to `fortify` spatial objects.
+Basically, this means, you need to combine the coordinates and attribute slots of a spatial class object into one dataframe.
+While this works well in the case of points, it duplicates the same information over and over again in the case of polygons, since each coordinate (vertex) of a polygon receives the attribute data of the polygon.
+This is especially disadvantageous if you need to deal with tens of thousands of polygons.
+With the introduction of simple features to R this limitation disappears, and it seems likely that this will make **ggplot2** the standard tool for the visualization of vector data. 
+This might be different regarding the visualization of raster data. Raster visualization methods received a boost with the release of **rasterVis** [@lamigueiro_displaying_2014] which builds on top of the lattice system.
+
+More recently, new packages aim at easing the creation of complex, high-quality maps with minimal code.
+The **tmap** package (released in 2014) might serve as an archetype for this kind of development [@R-tmap].
+It facilitates the user-friendly creation of thematic maps with an intuitive command-line interface  (see also [**mapmisc**](https://cran.r-project.org/package=mapmisc)) . 
 <!-- ADD THIS LATTER -->
 <!-- CITE the paper Tennekes, M. (2017) tmap: Thematic Maps in R. Forthcoming in the Journal
 of Statistical Software http://von-tijn.nl/tijn/research/presentations/tmap_user2017.pdf-->
-**tmap** provides legends, scale bars, grid lines, north arrows, optimized for mapping and integrates with the **leaflet** package (released in 2015) to offer interactive maps [@R-leaflet]. 
-Additionally, the **mapview** package was created on top of **leaflet** [@R-mapview], enabling interactive maps to be created rapidly from **sp** or **sf** objects using a range of background maps, scale bars and more.
+**tmap** provides legends, scale bars, grid lines, and north arrows optimized for mapping, and also works in harmony with the **leaflet** package (released in 2015) for an easy display of interactive maps [@R-leaflet]. 
```

Comment:
I suggest this (aiming to be more concise and clear):

```
**tmap** is a sophisticated yet user friendly mapping package which works in harmony with the **leaflet** package (released in 2015) for interactive map making [@R-leaflet]. 
```

We'll cover mapping in a later chapter so no need to go into details here - we should cross-ref that chapter when it's done though!